### PR TITLE
FRW-332 Change min PHP version to 8.0

### PR DESCRIPTION
**Pull request by geega**
Ticket: https://spryker.atlassian.net/browse/FRW-332

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]

**Review by dimitriyTsemma — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "homepage": "https://spryker.com",
     "license": "MIT",
     "require": {
-        "php": ">=7.4",
+        "mikey179/vfsstream": "^1.6.7",
```
Comment (dereuromark) [inline composer.json:8]: why are we adding this to require?

Comment (dereuromark) [reply composer.json:8]: I dont think this is a valid change

Comment (dereuromark) [reply composer.json:8]: Where in /src/ do you see this dependency being needed or used?
This is for testing harness, so it by design, cannot be a require dependency.

Comment (geega) [reply composer.json]: Yes. You are right. I moved this dependency to dev section and all good

**Review by dereuromark — APPROVED** [review]

**Pull request merged**